htdocs/core/tpl/objectline_create.tpl.php

```diff
@@ -1073,6 +1073,8 @@ function(data) {
 			} else { jQuery("#pbq").val(''); }
 			<?php
 		}
+		$reshook = $hookmanager->executeHooks('objectlinecreatesupplierorderproductselected', $parameters, $object, $action);
```

Comment:
As the name of hook is also a php function, can you use the camelCase syntax for the name ofthe hook...

Reply:
done